Add libpci3 pkg to docker-platform-monitor

VenkatCisco opened the pull request:
<!--
     Please make sure you've read and understood our contributing guidelines:
     https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

     ** Make sure all your commits include a signature generated with `git commit -s` **

     If this is a bug fix, make sure your description includes "fixes #xxxx", or
     "closes #xxxx" or "resolves #xxxx"

     Please provide the following information:
-->

#### Why I did it
 The libpci library provides portable access to configuration registers of devices connected to the PCI bus. 
#### How I did it
update dockers/docker-platform-monitor/Dockerfile.j2. Total usage is: 

root@sonic:/var/log#  dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | grep libpci3
102     libpci3

#### How to verify it
Verify PCI based devices are mounted successfully on the PCI bus 

#### Which release branch to backport (provide reason below if selected)

<!--
- Note we only backport fixes to a release branch, *not* features!
- Please also provide a reason for the backporting below.
- e.g.
- [x] 202006
-->

- [ ] 201811
- [ ] 201911
- [x] 202006
- [x] 202012

#### Description for the changelog

 The libpci library provides portable access to configuration registers of devices connected to the PCI bus.

```
root@sonic:/home/admin# dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | grep libpci3
102     libpci3
```


#### A picture of a cute animal (not mandatory but encouraged)

jleveque commented:
@VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?

VenkatCisco commented:
> @VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?

Done.

jleveque commented:
> > @VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?
> 
> Done.

Sorry, but I don't see it in the description.

VenkatCisco commented:
> #### Why I did it
> The libpci library provides portable access to configuration registers of devices connected to the PCI bus.

root@sonic:/var/log#  dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | grep libpci3
102     libpci3
> 
> #### How I did it
> update dockers/docker-platform-monitor/Dockerfile.j2
> 
> #### How to verify it
> Verify PCI based devices are mounted successfully on the PCI bus
> 
> #### Which release branch to backport (provide reason below if selected)
> * [ ]  201811
> * [x]  201911
> * [x]  202006
> * [x]  202012
> 
> #### Description for the changelog
> #### A picture of a cute animal (not mandatory but encouraged)

VenkatCisco commented:
> > > @VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?
> > 
> > 
> > Done.
> 
> Sorry, but I don't see it in the description.

somehow the details are hidden in the html tags i guess. I have added it to the top section as well.

jleveque commented:
> > > > @VenkatCisco: In the PR description, can you please indicate the installed size of this package on disk?
> > > 
> > > 
> > > Done.
> > 
> > 
> > Sorry, but I don't see it in the description.
> 
> somehow the details are hidden in the html tags i guess. I have added it to the top section as well.

Thanks. I have fixed the description here and in the other PRs.

VenkatCisco commented:
Thanks Joe. Really appreciate the quick response.


From: Joe LeVeque ***@***.***>
Date: Tuesday, June 29, 2021 at 11:39 AM
To: Azure/sonic-buildimage ***@***.***>
Cc: Venkat Garigipati (venkatg) ***@***.***>, Mention ***@***.***>
Subject: Re: [Azure/sonic-buildimage] add libpci3 pkg to docker-platform-monitor (#8016)

@VenkatCisco<https://github.com/VenkatCisco>: In the PR description, can you please indicate the installed size of this package on disk?

Done.

Sorry, but I don't see it in the description.

somehow the details are hidden in the html tags i guess. I have added it to the top section as well.

Thanks. I have fixed the description here and in the other PRs.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/sonic-buildimage/pull/8016#issuecomment-870826349>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASPBHRV3RTU7N2X5AEARTVDTVIHMPANCNFSM47QNNTHA>.

jleveque commented:
@VenkatCisco: Is it necessary to backport this to the 201911 branch? That branch is relatively stable; we only want to backport bug fixes.

VenkatCisco commented:
> @VenkatCisco: Is it necessary to backport this to the 201911 branch? That branch is relatively stable; we only want to backport bug fixes.

I updated the PR to reflect it. Our Primary development branch is 202012 and Azure/master.
